docs/react/adapters/solid-query.md

```diff
@@ -69,10 +69,10 @@ Solid Query offers an API similar to  React Query, but there are some key differ
 
 ```tsx
 // ❌ react version
-useQuery(["todos", todo], fetchTodos)
+useQuery({queryKey: ["todos", todo], queryFn: fetchTodos})
 
 // ✅ solid version
-createQuery(() => ["todos", todo()], fetchTodos)
+createQuery({queryKey: () => ["todos", todo()], queryFn: fetchTodos})
```

Comment:
I'm not sure if this is technically true - I think the whole object we pass into `createQuery` should be a function?

See the roadmap discussion with @ardeora here:
- https://github.com/TanStack/query/discussions/4252#discussioncomment-3779788

Also, for consistency, let's please use multi line whenever we use the object syntax:

```
useQuery({
  queryKey: ["todos", todo],
  queryFn: fetchTodos,
})
```

Reply:
Done ✅